controllers/memberstatus/memberstatus_controller_test.go

```diff
@@ -393,10 +412,11 @@ func TestOverallStatusCondition(t *testing.T) {
 				HasRoutes("https://console.member-cluster/console/", "https://codeready-codeready-workspaces-operator.member-cluster/che/", routesAvailable())
 		})
 
-		t.Run("when missing NodeMetrics for Node", func(t *testing.T) {
+		t.Run("when missing NodeMetrics for two Nodes", func(t *testing.T) {
 			// given
-			nodeAndMetrics := newNodesAndNodeMetrics(forNode("worker-123", []string{"worker"}, "3000000Ki"))
-			reconciler, req, fakeClient := prepareReconcile(t, requestName, getHostClusterFunc, allNamespacesCl, nodeAndMetrics[0], memberOperatorDeployment, memberStatus)
+			nodeAndMetrics1 := newNodesAndNodeMetrics(forNode("worker-123", []string{"worker"}, "3000000Ki"))
```

Comment:
Could we rename those vars to something like `unreadyNodeAndMetrics1` and `unreadyNodeAndMetrics2` and add a comment (as mentioned above)